npc/re/instances/saras_memory.txt

```diff
@@ -1089,22 +1089,22 @@ OnDisable:
 	end;
 OnRespawn:
 	.@map$ = instance_mapname("1@sara");
-	monster(.@map$, 150, 232, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("I am getting too old for this sh-"));
-	monster(.@map$, 150, 227, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
-	monster(.@map$, 150, 222, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
-	monster(.@map$, 150, 217, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
-	monster(.@map$, 161, 232, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
-	monster(.@map$, 161, 227, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
-	monster(.@map$, 161, 222, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
-	monster(.@map$, 161, 217, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
-	unittalk($@mobid[0], _("For Payon!!"));
+	'mobid = monster(.@map$, 150, 232, _("Payon Town Guard"), PAYONSOLDIER2, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
```

Comment:
I don't think we need an instance variable here. A local variable would be more than enough. (.@mobid)